src/components/Onboarding/Deployment/Boxes.js

```diff
@@ -259,6 +279,7 @@ export function BoxReady({ isFinalized, onGetStarted, opacity, boxTransform }) {
         >
           {isFinalized ? (
             <div>
+              <PageVisibility onChange={throwConfetti} />
```

Comment:
Wondering why we use `PageVisibility` here and not for the create proposal screen. Also is it totally necessary?

Reply:
The create proposal screen have the autoclose modal, so dont wokr the same way like the Onboardind wait for the user interact to close the modal.

Reply:
What type of interaction is required in this case to throw the conffeti? Isn´t it thrown when the garden deployment si finalized as stated in the useEffect logic?

Reply:
Why use page visibility ? 
Why signing is slow the whole process take time and confirm transactions, and you might been navigating on another tab of browser and just when your Wallet ask, you confirm signing. Or you are back and forth and if it finish and you not there you miss the confetti.

The use case for it was my self, while i was testing i miss the confetti because MetaMask ask to sign and I don’t need be there on that page (tab) to confirm.

So the use the PageVisibility its improve to UX, if yours think its not necessary i can remove it. @fabriziovigevani @0xGabi

Reply:
Hey @kamikazebr what do you think of instead throwing the confetti on the `onClick` event of the `Get started` button a few lines below? 

It is the last step of the whole onboarding process seems the right time and we can remove the `react-page-visibility`.

Reply:
Nice idea! I'll do that :D